src/renderer/features/operations/OperationsConfirm/Nominate/ui/Confirmation.tsx

```diff
@@ -136,7 +137,7 @@ export const Confirmation = ({ secondaryActionButton, onGoBack }: Props) => {
               onClick={toggleValidators}
             >
               <div className="rounded-[30px] px-1.5 py-[1px] bg-icon-accent">
-                <CaptionText className="text-white">{confirmStore.validators.length}</CaptionText>
+                <CaptionText className="text-white">{confirmStore.validators?.length}</CaptionText>
```

Comment:
I think there should be either ternary check around `<CaptionText>` node or some default value, like `confirmStore.validators?.length ?? 0`